cmd/repl/repl_test.go

```diff
@@ -36,6 +39,44 @@ import (
 
 var space = regexp.MustCompile(`\s+`)
 
+func testMainFastFail(t *testing.T, interactive bool) {
+	a := assert.New(t)
+	// Run the crashing code when FLAG is set
+	if os.Getenv("FLAG") == "false" {
```

Comment:
> Issue #1338 is about SQLFlow doesn't present a friendly error message when DB connection fails. I don't understand why we could fix this issue by adding a unit test?

The bug in #1338 occurs since the original `repl.go` doesn't check the connectability of DB before entering the interactive mode. The fix is actually the new  function `assertConnectable` in `repl.go`. and the call to it in the function `main`

Reply:
> For less probability of name conflicting, I assume we'd change `FLAG` to a longer and more precise name like `SQLFLOW_TEST_FAST_FAIL_INTERACTIVE_OR_NOT`.

Done.